cli/src/test/java/hudson/cli/PrivateKeyProviderTest.java

```diff
@@ -84,21 +89,21 @@ public void loadKeyOpenSSH() throws IOException, GeneralSecurityException {
     /**
     key command: ssh-keygen -f openssh-unsupported -t rsa -b 1024 -p password
     */
-    @Test(expected = NoSuchAlgorithmException.class)
+    @Test
```

Comment:
is this change required by the new jUnit version? Because it may break a lot of tests in this case.

Reply:
it's replaced by the `assertThrows`